cylc/flow/task_pool.py

```diff
@@ -1158,15 +1162,18 @@ def _reload_taskdefs(self) -> None:
                 self._swap_out(new_task)
                 self.data_store_mgr.delta_task_prerequisite(new_task)
                 LOG.info(f"[{itask}] reloaded task definition")
+
                 if itask.state(*TASK_STATUSES_ACTIVE):
-                    LOG.warning(
-                        f"[{itask}] active with pre-reload settings"
-                    )
+                    warn_tasks.append(str(itask))
                 elif itask.state(TASK_STATUS_PREPARING):
                     # Job file might have been written at this point?
-                    LOG.warning(
```

Comment:
Agreed.

* This is what the user told Cylc to do.
* The user does not need to review this list of tasks.
* There is no call to action here.